### PR TITLE
references: add DM v1.0.2 release notes

### PR DESCRIPTION
**Pull request by csuzhangxc**
### What is changed, added or deleted? 

add DM v1.0.2 release notes

### Which version does your change affect? 

dev, v3.0 and v2.1

### CODE REVIEW

**Comment by csuzhangxc** [comment]
@yikeke @amyangfei PTAL

File: dev/reference/tools/data-migration/releases/1.0.2.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+# DM 1.0.2 Release Notes
+
+发版日期：2019 年 10 月 30 日
+
+DM 版本：1.0.2
+
+DM-Ansible 版本：1.0.2
+
+## 提升
```
Comment (yikeke) [inline dev/reference/tools/data-migration/releases/1.0.2.md]:
```suggestion
## 改进提升
```
同 TiDB 的 release 统一一下用词，参见[TiDB 3.0.4 Release Notes](https://pingcap.netlify.com/docs-cn/v3.0/releases/3.0.4/)

File: dev/reference/tools/data-migration/releases/1.0.2.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+- 支持自动为 DM-worker 生成部分配置项
+- 支持自动为数据迁移任务生成部分配置项
+- 简化 `query-status` 在无参数时的默认输出
+- DM 直接管理到下游数据库的连接
+
+## Bug 修复
```
Comment (yikeke) [inline dev/reference/tools/data-migration/releases/1.0.2.md]:
```suggestion
## 问题修复
```
同上

File: dev/reference/tools/data-migration/releases/1.0.2.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+- 修复在进程启动过程中以及执行 SQL 失败时可能 panic 的问题
+- 修复 DDL 执行超时后可能造成 sharding DDL 协调异常的问题
+- 修复由于前置检查超时或部分 DM-worker 不可访问而不能启动数据迁移任务的问题
+- 修复 SQL 执行失败后可能错误重试的问题
+
+## 详细变更与 bug 修复
```
Comment (yikeke) [inline dev/reference/tools/data-migration/releases/1.0.2.md]:
```suggestion
## 详细变更及问题修复
```

**Comment by csuzhangxc** [comment]
@yikeke PTAL again

**Pull request merged**